gk/main.c

```diff
@@ -548,41 +549,35 @@ gk_del_flow_entry_from_hash(struct rte_hash *h, struct flow_entry *fe)
 }
 
 static void
-gk_flow_tbl_bucket_scan(uint32_t *bidx,
+gk_flow_tbl_entry_scan(uint32_t entry_idx,
 	uint64_t request_timeout_cycles, struct gk_instance *instance)
 {
-	int32_t index;
-	const struct ip_flow *key;
-	void *data;
-	uint32_t next = 0;
-	uint64_t now = rte_rdtsc();
+	struct flow_entry *fe = &instance->ip_flow_entry_table[entry_idx];
+	uint64_t now;
```

Comment:
Could add the line `rte_prefetch_non_temporal(fe);` and check if it affects the performance?

Reply:
I added this line and ran the same tests, and the results are virtually the same.

Reply:
Thank you for testing it.